pymc/distributions/multivariate.py

```diff
@@ -265,6 +265,13 @@ class MvStudentTRV(RandomVariable):
     dtype = "floatX"
     _print_name = ("MvStudentT", "\\operatorname{MvStudentT}")
 
+    def make_node(self, rng, size, dtype, nu, mu, cov):
+        nu = at.as_tensor_variable(floatX(nu))
```

Comment:
```suggestion
        nu = at.as_tensor_variable(nu)
```
I think we don't need the `floatX` here, that is something that belongs to PyMC distributions.